packages/jet-brains-integration/src/web-types-generator.ts

```diff
@@ -55,49 +56,73 @@ export function getTagList(
           description: slot.description,
         };
       }),
-      events: getWebTypeEvents(component),
```

Comment:
When viewing the final Web Types JSON file with this property included, the schema warns that it is deprecated. Just wanted to point out that I removed this in case it is still needed for something.

Reply:
I think we may want to keep this for now in case devs have older editors that still may be using it.

Reply:
Fair point, though in that case I would suggest adding an option to the plugin config that allows users to target a specific Web Types schema version in case it is fully removed in the future and causes errors in newer editors. That's unrelated to the purpose of this PR though, so I will revert it here.

Reply:
Yeah, I can chat with the JetBrains team to see what their plans are.